src/core/scene/a-scene.js

```diff
@@ -56,7 +56,13 @@ module.exports = registerElement('a-scene', {
 
     init: {
       value: function () {
-        this.behaviors = [];
+        this.behaviors = {
+          tick: [],
+          tock: [],
+          tickPostRender: [],
+          tockPostRender: []
+        };
+        this.postNeedsNotify = false;
```

Comment:
Well this will lead to several events handled where one would be sufficient, usually you'll pass several components to setup the chain at once. The compositor will catch the events and reconfigure itself. It's not really a problem as this will generally not happen often but If you mean to generalize the notification events for all types of behaviors then this will probably lead to alot of event traffic.